websites/S/Stadia/dist/metadata.json

```diff
@@ -4,6 +4,12 @@
     "name": "Alexx",
     "id": "604453403640463371"
   },
+  "contributors": [
+    {
+      "name": "Pjoottr",
+      "id": "906309000465903697"
+    }
+  ],
```

Comment:
![image](https://user-images.githubusercontent.com/54318514/145173549-92854670-ccc6-42e7-a905-a99d91129331.png)
Not exactly a quarter yet if you add support for library path then sure